synapse/config/workers.py

```diff
@@ -17,9 +17,28 @@
 
 import attr
 
-from ._base import Config, ConfigError, ShardedWorkerHandlingConfig
+from ._base import (
+    Config,
+    ConfigError,
+    RoutableShardedWorkerHandlingConfig,
+    ShardedWorkerHandlingConfig,
+)
 from .server import ListenerConfig, parse_listener_def
 
+_FEDERATION_SENDER_WITH_SEND_FEDERATION_ENABLED_ERROR = """
+The send_federation must be disabled in the main synapse process
```

Comment:
I feel like these are both missing a word -- maybe "The send_federation *setting*..."?